src/coreComponents/constitutive/contact/HydraulicApertureTable.hpp

```diff
@@ -156,10 +157,11 @@ GEOS_HOST_DEVICE
 GEOS_FORCE_INLINE
 real64 HydraulicApertureTableUpdates::computeHydraulicAperture( real64 const aperture,
                                                                 real64 const normalTraction,
+                                                                integer const fractureState,
```

Comment:
```suggestion
                                                                FractureState const fractureState,
```

Reply:
keep it as integer to avoid compilation error